package.json

```diff
@@ -45,29 +45,29 @@
     "test:watch": "jest --watch"
   },
   "dependencies": {
-    "gatsby": "next",
-    "gatsby-plugin-catch-links": "next",
-    "gatsby-plugin-feed": "next",
-    "gatsby-plugin-google-analytics": "next",
+    "gatsby": "^2.0.0-rc.21",
```

Comment:
Don't really want to go back to `next` since Yarn upgrade change it alone. I also prefer to see the version number.